File: ruby/raindrops/raindrops.rb
class Raindrops

  def self.convert(number)
    raindrop = pling(number) + plang(number) + plong(number)
    if raindrop == '' then number.to_s
    else raindrop
    end
  end

  def self.pling(number)
    (number % 3).zero? ? 'Pling' : ''
  end

  def self.plang(number)
    (number % 5).zero? ? 'Plang' : ''
  end

  def self.plong(number)
    (number % 7).zero? ? 'Plong' : ''
  end

  end

module BookKeeping
  VERSION = 3 # Where the version number matches the one in the test.
end

File: python/raindrops/raindrops.py
raindrop = {3: 'Pling', 5: 'Plang', 7: 'Plong'}


def convert(number: int) -> str:
    output_list = [raindrop[key] for key in raindrop if number % key == 0]
    output = "".join(output_list)
    return str(number) if output == '' else output
